lib/tasks/notify_report.rake

```diff
@@ -0,0 +1,6 @@
+namespace :report do
+  desc "Query the Notify API for email(s) by reference"
+  task :get_notifications_from_notify, [:reference] => :environment do |_t, args|
```

Comment:
I wonder if we could have another task which takes an email ID, finds each delivery attempt for it and prints the report for each one of those.